### DISCUSSION TITLE
Capitolo 5 - Errore nome costante listato 5.3

### DISCUSSION THREAD

**Discussion by NersOfficial** [lifeisfoo/react-guida-pratica#4] [Errori e refusi]
Nel codice del ` Listato 5.3 ` è presente un errore. Precisamente, la costante che contiene il nome delle classi degli elementi `<li>` non corrisponde a quella usata dentro l'istruzione `return()`.

```jsx
>>  const itemClasses = "list-group-item border rounded-1 my-1";

export default function TodoItem({ done, text }) {
  const textClasses = `mx-1 my-0 ps-3 flex-grow-1 ${done ? "text-decoration-line-through text-black-50" : ""}`;
   
  return (
>>    <li className={todoClasses}>
      <input  className="form-check-input mx-1 my-0"  type="checkbox" checked={done} />
      <p className={textClasses}>{text}</p>
    </li>
  );
}

``` 
Riga 1: Nome costante `itemClasses` invece di  `todoClasses`;
Riga 7: Nome usato nell'interpolazione `todoClasses`;

Ps: nei restanti esempi l'errore non è presente.

**Comment by lifeisfoo** [comment]
Grazie per la segnalazione!

**Reply by NersOfficial** [reply]
Di nulla.
